examples/CMakeLists.txt

```diff
@@ -112,6 +112,10 @@ IF(DEAL_II_COMPONENT_EXAMPLES)
               )
           ENDIF()
 
+          IF(CMAKE_VERSION VERSION_GREATER_EQUAL 3.18)
```

Comment:
Why do you need to do this here and not inside the individual examples? Is it possible to put this inside one of the setup_target cmake macros?